playbook/app/pb_kits/playbook/pb_text_input/add_on.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbTextInput
+    class AddOn < Playbook::KitBase
+      prop :icon, type: Playbook::Props::String, required: true
+      prop :alignment, type: Playbook::Props::Enum, values: %w[right left], default: "right"
+      prop :border, type: Playbook::Props::Boolean, default: true
+
+      def border_css
+        border_toggle = border ? 'on' : 'off'
+        if alignment == 'left'
+          border_to_change = 'right'
+        else
+          border_to_change = 'left'
+        end
+
+        "border_#{border_to_change}_#{border_toggle}"
+      end
+
+      def dark_mode_css
+        dark ? 'add-on-card-dark' : nil
+      end
```

Comment:
Perhaps that method could live here?

Reply:
@thestephenmarshall I believe I have implemented all your suggested changes, except perhaps this one, which I'm not 100% sure what you mean. I did make a change....did you mean something like that or something else?